Consumer user can be created in Azure Portal now (#46678)
Azure/azure-content: articles/active-directory-b2c/active-directory-b2c-faqs.md

In an Azure AD B2C tenant, most apps want the user to sign-in with any arbitrary email address (for example, joe@comcast.net, bob@gmail.com, sarah@contoso.com, or jim@live.com). This type of account is a local account. We also support arbitrary user names as local accounts (for example, joe, bob, sarah, or jim). You can choose one of these two local account types when configuring identity providers for Azure AD B2C in the Azure portal. In your Azure AD B2C tenant, select **Identity providers**, select **Local account**, and then select **Username**.

> User accounts for applications must always be created through a sign-up user flow, sign-up or sign-in user flow, or by using the Azure AD Graph API. Each User accounts can be created in the Azure portal.

Reviewer: Good catch, @watahani! To simplify it further, how about this?

```suggestion
User accounts for applications can be created through a sign-up user flow, sign-up or sign-in user flow, the Microsoft Graph API, or in the Azure portal.
```

Author: Thank you for your feedback. It looks good for me.

Author: It seems that I have no permission to commit suggestion so @mmacy could you please commit your suggestion?

Reviewer: Hi @watahani, if you merge this PR into your fork, that should do it: https://github.com/watahani/azure-docs/pull/1

(I don't have write perms to the MicrosoftDocs/azure-docs repo, either, so we'll need to do it this way.)
